### DISCUSSION TITLE
Future cron job for recurring Transactions

### DISCUSSION THREAD

**Discussion by jeremywall-ww** [firefly-iii/firefly-iii#6703] [Firefly III]
Hi, just wondering if you have any advice? I want to run the cron with a future date eg 31 Dec 2023 and I would ideally like FF3 to create **all** transactions from today up until that period, but it seems that the --force and --date in the cron job only runs the cron for transactions that fall on that date, and not any inbetween. Is there any way around this?

**Comment by JC5** [comment]
No, you'll have to hit each day individually. But you can run the command as often as you like.
